samples/hubspot/program.py

```diff
@@ -0,0 +1,38 @@
+"""Demonstration of AutoKitteh's HubSpot integration.
+
+This script showcases two basic operations with the HubSpot API:
+creating a new contact and listing all deals.
+"""
+
+from autokitteh.hubspot import hubspot_client
+from hubspot.crm.contacts import SimplePublicObjectInputForCreate
+from hubspot.crm.contacts.exceptions import ApiException
+
+
+hubspot = hubspot_client("hubspot_conn")
+
+
+def create_contact(event):
+    contact_properties = {
+        "email": "meow@autokitteh.com",
+        "firstname": "Kitty",
+        "lastname": "Meowington",
+    }
+    try:
+        contact_input = SimplePublicObjectInputForCreate(properties=contact_properties)
+
+        # Send a request to HubSpot to create the contact
```

Comment:
1. The code below this comment is trivial, so this comment is unnecessary, it doesn't add any information to the reader
2. If you would have kept this comment, I'd ask to add "." at the end